packages/odyssey-react-mui/src/Toast.tsx

```diff
@@ -0,0 +1,155 @@
+/*!
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { AlertColor } from "@mui/material";
+import React, {
+  useContext,
+  useEffect,
+  memo,
+  forwardRef,
+  ForwardedRef,
+} from "react";
+import {
+  Alert,
+  AlertTitle,
+  Button,
+  CloseIcon,
+  Link,
+  Snackbar,
+  visuallyHidden,
+} from ".";
+import { ToastStackContext } from "./ToastStackContext";
+
+export type ToastProps = {
+  /**
+   * If set, this determines how long the toast should appear before automatically
+   * disappearing, in milliseconds. It will only take effect if the toast is not dismissable.
+   * If left blank, it defaults to 6000.
+   */
+  autoHideDuration?: number;
+  /**
+   * If isDismissable is true, the alert will include a close button
+   */
+  isDismissable?: boolean;
+  /**
+   * If true, the Toast is visible
+   */
+  isOpen?: boolean;
+  /**
+   * If isStatic is true, the alert will not respect Snackbar positioning
```

Comment:
```suggestion
   * If true, the alert will not respect `Snackbar` positioning
```

Reply:
Is this a prop we want on the `Toast`? Don't we have `Infobox` and `Banner` for inline stuff?

Reply:
This is a fix for an obnoxious problem we have.

`Toast` needs to include `Snackbar`, since that's what controls our show/hide logic and the auto-dismiss timer. However, `Snackbar` is always fixed to the edge of the screen. I built in `isStatic` as a way to disable the positioning, so `Toast` will respect its container's positioning.

If the consumer is using a single `Toast`, they can leave `isStatic` as `false`, since they'll probably want the Snackbar positioning intact.

If the consumer is using `Toast` within a `ToastStack`, they'll want to use the `ToastStack`'s positioning, in which case they should set `isStatic` to `true`.

On second thought, maybe we should remove this as a prop and handle it entirely with context 🤔

Reply:
Could/should we default to using `ToastStack`?